java/junit.ui/src/org/netbeans/modules/junit/ui/resources/JUnit5TestClass.java.template

```diff
@@ -0,0 +1,67 @@
+<#-- FreeMarker template (see http://freemarker.org/) -->
```

Comment:
Please add the template file to the [licenseinfo.xml](https://github.com/apache/incubator-netbeans/blob/master/java/junit.ui/licenseinfo.xml).

Reply:
same comment as above.